bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -119,6 +120,7 @@
 
     final AtomicInteger blockAddCompletions = new AtomicInteger(0);
     final AtomicInteger numEnsembleChanges = new AtomicInteger(0);
+    final int maxNumEnsembleChanges;
```

Comment:
maxAllowedEnsembleChanges may be better variable

Reply:
good, fixed